Ticket - 28567 Change to O-R-5

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files -oerep/r-bookedN.i,oerep/r-bookedN.w

sewa67 committed:
Ticket - 28567 Change to O-R-5
Files -oerep/r-bookedN.i,oerep/r-bookedN.w

brad-advantzware reviewed (changes requested):
I would prefer these be in a common program that returns: "GetRoutingForJob"  "GetInksForJob".  Also, I don't see the AOA version of OR5 changed.

sewa67 committed:
Ticket - 28567 Change to O-R-5
Files - ao/BL/r-booked.i,aoa,tempTable/ttOrdersBooked.i,oerep/r-bookedN.i,oerep/r-bookedN.w
